src/System.Memory/src/System/Buffers/ArrayMemoryPool.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+#if !netstandard
+using Internal.Runtime.CompilerServices;
+#else
+using System.Runtime.CompilerServices;
+#endif
+
+namespace System.Buffers
+{
+    internal sealed partial class ArrayMemoryPool<T> : MemoryPool<T>
+    {
+        private const int s_maxBufferSize = int.MaxValue;
```

Comment:
Nit: s_ prefix is used for statics; consts should be PascalCased.

Reply:
I know - this is kinda tough case since the "best" name is already taken by the property. <b>MaxBufferSizeConst</b> just sounds awkward.